static/app/views/releases/list/index.tsx

```diff
@@ -325,18 +354,38 @@ export default function ReleasesList() {
     <PageFiltersContainer showAbsolute={false} defaultSelection={selection}>
       <SentryDocumentTitle title={t('Releases')} orgSlug={organization.slug} />
       <NoProjectMessage organization={organization}>
-        <Header />
-        <Layout.Body>
-          <Layout.Main width="full">
-            <ReleaseHealthCTA
-              organization={organization}
-              releases={releases}
-              selectedProject={selectedProject}
-              selection={selection}
-            />
-            <ReleasesPageFilterBar condensed>
+        <Layout.Header noActionWrap>
```

Comment:
Inlines the header now and moves the project selector into it. This is intentional to give us the divider under the tabs.

<img width="1654" height="664" alt="Image" src="https://github.com/user-attachments/assets/778aacee-70c5-401a-b24e-2a50e5fd0590" />